.changeset/nasty-oranges-heal.md

```diff
@@ -0,0 +1,5 @@
+---
+'@launchpad-ui/progress-bubbles': patch
```

Comment:
```suggestion
'@launchpad-ui/progress-bubbles': patch
'@launchpad-ui/core': patch
```
We always include `core` to ensure that package gets the changes

Reply:
Ah! I see, thanks!